### PR TITLE
replaced static compiler.jar with dependencies to google-closure-compiler npm package

### PR DESCRIPTION
**Pull request by simone-sanfratello**
please update publish to npm

### CODE REVIEW

**Comment by tim-smart** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -6,8 +6,10 @@
   { "type"      : "git"
   , "url"       : "git://github.com/tim-smart/node-closure.git"
   }
+, "dependencies": {
+    "google-closure-compiler": "x"
```
Comment (jdalton) [inline package.json]: version `x`? That doesn't sound like a good idea.

Comment (jdalton) [inline package.json]: Should probably [lock to a specific version](https://www.npmjs.com/package/google-closure-compiler) per update so the minifier (and possible bugs changes) don't change under this package's feet.

Comment (tim-smart) [inline package.json]: You are right.

Main issue would be between deployment of developers apps. This library doesn't have any google closure specific code, it just passes around arguments from the developer.

Might look into something like `peerDependencies`

Comment (jdalton) [inline package.json]: Before the [jar was included](https://github.com/tim-smart/node-closure/commit/e8d5fc8393361751619eb9cbf68e9f27284d66d4) so versions were locked to whatever was included. Now that it's an external dep, `20150901.0.0`, you can still lock to a version.

It should also be noted that despite the name, [google-closure-compiler](https://www.npmjs.com/package/google-closure-compiler), is not maintained or managed by Google but rather some rando dude.

**Comment by simone-sanfratello** [comment]
@jdalton but the target of the X version is to get the last version of the compiler, when the random dude update from google, without have to update manually, just using npm update to get all updated

**Comment by jdalton** [comment]
The problem is bugs often slip into updated minifiers which is why I lock versions. Now this will update under foot which is kind of lame and a regression from the previous behavior.

**Comment by tim-smart** [comment]
The latest version has locked it down, if you look at the latest
package.json!

**Comment by jdalton** [comment]
:rocket: !

**Pull request merged**